src/main/java/com/redhat/labs/lodestar/model/EngagementUser.java

```diff
@@ -25,4 +25,29 @@
     @NotBlank
     private String role;
 
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
+        EngagementUser other = (EngagementUser) obj;
+        if (email == null) {
+            if (other.email != null)
+                return false;
+        } else if (!email.equals(other.email))
+            return false;
+        return true;
+    }
```

Comment:
Wouldn't this last section be easier to read with
```
if(email == null && other.email != null) {
  return false;
}
return email.equals(other.email);

Reply:
cleaned up code based on comment.